src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -178,37 +176,9 @@ func onBootstrapControlPlane(s *state.State, initConfig map[string]string) error
 		return fmt.Errorf("failed to get IP address(es) from ServiceCIDR %q: %w", cfg.Network.ServiceCIDR, err)
 	}
 
-	switch cfg.APIServer.Datastore {
-	case "k8s-dqlite":
-		certificates := pki.NewK8sDqlitePKI(pki.K8sDqlitePKIOpts{
-			Hostname:          s.Name(),
-			IPSANs:            []net.IP{{127, 0, 0, 1}},
-			Years:             20,
-			AllowSelfSignedCA: true,
-		})
-		if err := certificates.CompleteCertificates(); err != nil {
-			return fmt.Errorf("failed to initialize cluster certificates: %w", err)
-		}
-		if err := setup.EnsureK8sDqlitePKI(snap, certificates); err != nil {
-			return fmt.Errorf("failed to write cluster certificates: %w", err)
-		}
-
-		cfg.Certificates.K8sDqliteCert = certificates.K8sDqliteCert
-		cfg.Certificates.K8sDqliteKey = certificates.K8sDqliteKey
```

Comment:
NOTE: this is now missing in the bootstrap code, which is also why the e2e tests are failing. perhaps you can add it after the `setupDatastoreCertificates`?

Reply:
Ah thanks for catching that for me!